frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -92,6 +92,7 @@ export enum FLAGS {
   CONSOLE_CAPABILITY_LIGHTSPEEDBUTTON_IS_ENABLED = 'CONSOLE_CAPABILITY_LIGHTSPEEDBUTTON_IS_ENABLED',
   CONSOLE_CAPABILITY_GETTINGSTARTEDBANNER_IS_ENABLED = 'CONSOLE_CAPABILITY_GETTINGSTARTEDBANNER_IS_ENABLED',
   LIGHTSPEED_IS_AVAILABLE_TO_INSTALL = 'LIGHTSPEED_IS_AVAILABLE_TO_INSTALL',
+  DEVCONSOLE_PROXY_AVAILABLE = 'DEVCONSOLE_PROXY_AVAILABLE',
```

Comment:
nitpick: I'd just call this `DEVCONSOLE_PROXY`. Whether it is available is implied by the flag value (true or false)

```suggestion
  DEVCONSOLE_PROXY = 'DEVCONSOLE_PROXY',
```

Reply:
updated it.